src/components/Governance/EventsTable.tsx

```diff
@@ -1,122 +1,77 @@
-import { Grid, List, makeStyles, Typography } from '@material-ui/core';
-import { TimelockEvent } from 'mobx/model/governance-timelock/timelock-event';
-import { useCallback, useEffect, useMemo, useState } from 'react';
+import {
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TablePagination,
+  TableRow,
+} from '@material-ui/core';
+import { Skeleton } from '@material-ui/lab';
+import { GovernancePortalStore } from 'mobx/stores/GovernancePortalStore';
+import { observer } from 'mobx-react-lite';
 
 import EventsTableItem from './EventsTableItem';
-import { Pagination } from './Pagination';
-
-const useStyles = makeStyles((theme) => ({
-  infoPaper: {
-    paddingTop: theme.spacing(2),
-    display: 'flex',
-    justifyContent: 'flex-start',
-    alignItems: 'flex-start',
-    flexDirection: 'column',
-    maxHeight: '70vh',
-    width: '100%',
-    overflowX: 'auto',
-  },
-  root: {
-    padding: theme.spacing(1),
-    height: theme.spacing(6),
-    // width: '1100px',
-    textAlign: 'center',
-    borderTopLeftRadius: theme.shape.borderRadius,
-    borderTopRightRadius: theme.shape.borderRadius,
-    background: `${theme.palette.background.paper}`,
-  },
-  list: {
-    // width: '1100px',
-    overflowY: 'scroll',
-    background: `${theme.palette.background.paper}`,
-    padding: 0,
-    boxShadow: theme.shadows[1],
-  },
-}));
 
 export interface EventTableProps {
-  events?: Map<string, TimelockEvent>;
-  filters: string[];
+  governancePortal: GovernancePortalStore;
+  nextPage: (val: number) => void;
+  loadingProposals: boolean;
+  setPerPage: (val: number) => void;
 }
 
-const EventsTable = ({ events, filters }: EventTableProps): JSX.Element => {
-  const classes = useStyles();
-  const [eventListShow, setEventListShow] = useState<TimelockEvent[]>([]);
```

Comment:
lets remove the file: timelock-event.ts with the TimelockEvent so its not confused in the future as we shouldn't need it anymore